flexmeasures/data/migrations/versions/cb8df44ebda5_flexcontext_field.py

```diff
@@ -333,7 +335,7 @@ def downgrade():
         else:
             market_id = None
 
-        capacity_in_mw = (
+        site_power_capacity = (
             flex_context.get("site-power-capacity")
             if flex_context.get("site-power-capacity")
             else None
```

Comment:
This seems equivalent to:
```suggestion
            flex_context.get("site-power-capacity")
```

Reply:
Likewise for the other fields.

Reply:
I'm looking at this again and I'm wondering why I rote it like this in the first place, I went through the file history and its been like this since day 1 it seems 😅